data/remnant/remnant missions.txt

```diff
@@ -940,6 +940,143 @@ mission "Remnant: Learn Sign Follow Up"
 
 
 
+mission "Remnant: Follow-up on Terminus Researchers A"
+	name "Check up on Wormhole Researchers"
+	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
+	minor
+	source
+		government Remnant
+	destination Bounty
+	to offer
```

Comment:
These current conditions make it such that you could potentially get this mission to offer while you are researching the Ember Waste for the Deep scientists. You should probably add `not "Deep: Remnant 0: offered"`.

Reply:
Done. Thanks!

Reply:
OK, so this looks like it is now done